doc/configuration.md

```diff
@@ -50,6 +50,14 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_id_** (integer): the user ID of the owner.
     * **_group_** (object): specifies the group of the owner.
       * **_id_** (integer): the group ID of the owner.
+  * **_directories_** (list of objects): the list of directories to be created.
+    * **filesystem** (string): the internal identifier of the filesystem to create the directory in. This matches the last filesystem with the given identifier.
```

Comment:
Passive voice. "the internal identifier of the filesystem in which to create the directory."